app/views/posts/show.html.erb

```diff
@@ -1,51 +1,36 @@
-<div class="user_container">
-  <section class="post_show_cont">
-  <article class="post_show">
-    <div class="post_show_field">
-      <h4 class="post_show_title"> <%= @post.title %> by <%= @user.name %> </h4>
+<div class="container mt-5">
+  <section>
+    <article class="border border-secondary border-3 p-4">
+      <div class="d-flex align-items-center justify-content-between">
+        <h4 class="fs-5"> <%= @post.title %> by <%= @user.name %> </h4>
 
-      <p class="span_field"> Comments : <%= @post.comments.size %> Likes : <%= @post.likes.size %> </p>
-    </div>
+        <p class="fs-5 text-secondary"> Comments : <span class="fw-bolder"><%= @post.comments_counter || 0 %></span>, Likes : <span class="fw-bolder"><%= @post.likes_counter || 0 %></span></p>
+      </div>
 
-    <div>
-      <p> <%= @post.text %> </p>
-    </div>
-  </article>
+      <div class="mt-3">
+        <p> <%= @post.text %> </p>
+      </div>
+    </article>
 
-  <article class="span_comment_field">
-    <div>
-      <% @post.five_most_recent_comments.each do |comment| %>
+    <article>
       <div>
-        <li> <%= comment.users.name %>: <%= comment.text %> </li>
+        <%= form_with url: "#{@post.id}/likes" do |form| %>
+          <%= form.submit "Like" %>
+        <% end %>
```

Comment:
- [ ] The same thing for the like button. When I click on the like button, the following error appears:

<img width="406" alt="image" src="https://user-images.githubusercontent.com/77190203/173161397-ce1a6642-2018-45d5-a7dd-e1bc6d44baf2.png">

Please correct the routing error.

Reply:
including this @medaminedev66